growth/too/tasks/scheduler.py

```diff
@@ -225,3 +230,82 @@ def get_decam_dict(data_row, queue_name, cnt, nrows,
     decam_dict["wait"] = "False"
 
     return decam_dict
+
+def get_growthindia_table(json_data, sunrise_horizon=-12, horizon=20,
+                          priority=10000, domesleep=100):
+    """Make .csv file in GIT toO format for a given .json file"""
+    t = Table(rows=json_data['targets'])
+    coords = SkyCoord(ra=t['ra'], dec=t['dec'], unit=(u.degree, u.degree))
+    hanle = EarthLocation(lat=32.77889*u.degree,
+                          lon=78.96472*u.degree,
+                          height=4500*u.m)
+    iao = Observer(location=hanle, name="GIT",timezone="Asia/Kolkata")
+
+    twilight_prime = iao.sun_rise_time(Time.now(),
+                                       which="next",
+                                       horizon=sunrise_horizon*u.degree)\
+        - 12*u.hour
+    targets_rise_time = iao.target_rise_time(twilight_prime,
+                                             coords,
+                                             which="nearest",
+                                             horizon=horizon*u.degree)
+    targets_set_time = iao.target_set_time(targets_rise_time, coords,
+                                           which="next",
+                                           horizon=horizon*u.degree)
+    rise_time_IST = (targets_rise_time + 5.5*u.hour).isot
+    set_time_IST = (targets_set_time + 5.5*u.hour).isot
+    tend = targets_set_time
+    mooncoords = get_moon(tend, hanle)
+    sep = mooncoords.separation(coords)
+    dic = {}
+    dic['x'] = ['']*len(t)
+    dic['u'] = ['']*len(t)
+    dic['g'] = ['']*len(t)
+    dic['r'] = ['']*len(t)
+    dic['i'] = ['']*len(t)
+    dic['z'] = ['']*len(t)
+    target = ['EMGW']*len(t)
+    bands = {1: 'g', 2: 'r', 3: 'i', 4: 'z', 5: 'u'}
+    
+    for i in range(len(t)):
+        filt = bands[t[i]['filter_id']]
+        dic[filt][i] = '1X%i'%(t[i]['exposure_time'])
+
+    t.remove_columns(['request_id','filter_id','program_pi','program_id','subprogram_name','exposure_time'])
+    t.rename_column('field_id','tile_id')
+    t.rename_column('dec','Dec') 
+
+    domesleeparr = np.zeros(len(t)) + domesleep
+    priority = np.zeros(len(t)) + priority
+    minalt = AltitudeConstraint(min=horizon*u.degree)
+    always_up = is_always_observable(minalt,iao,coords,
+                                     Time(twilight_prime,
+                                          twilight_prime+12*u.hour))
+    rise_time_IST[np.where(always_up)] = (twilight_prime + 5.5*u.hour).isot
+    set_time_IST[np.where(always_up)] = (twilight_prime + 24*u.hour + 5.5*u.hour).isot
+    
+    ras_format = []
+    decs_format = []
+    for i in range(len(t)):
```

Comment:
`coords.to_string(...)` will return a list of strings, so you don't need to have an explicit `for` loop.